postman-collections/PAPI/postman-application-request/README.md

```diff
@@ -0,0 +1,30 @@
+# postman-application-request
+postman-application-request
+
+## Purpose
+This postman collection include all requests needed to perform an Application request workflow demo
```

Comment:
Can you remove the word "demo." This shouldn't really be a demo but more of an example .